sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobSasImplUtil.java

```diff
@@ -306,6 +330,7 @@ private String stringToSign(String canonicalName) {
             VERSION,
             resource,
             versionSegment == null ? "" : versionSegment,
+            this.encryptionScope == null ? "" : this.encryptionScope,
```

Comment:
when string to sign format changes we need to add new branch here.

Reply:
Do we not force SAS generation to be on latest version?

Reply:
we do, however there's a way to override it globally https://github.com/Azure/azure-sdk-for-java/blob/ad3586aef64218360521d72848f2befe6b7f5971/sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java#L88-L91 .

Reply:
Discussed offline apparently we no longer force this.